src/st/np3/3DEF4.c

```diff
@@ -442,13 +442,13 @@ void EntityEquipItemDrop(Entity* self) {
 
         g_api.PlaySfx(NA_SE_PL_IT_PICKUP);
 
-        if (itemId < 169) {
+        if (itemId < ITEM_HAND_END) {
```

Comment:
The two enum strategy seems to work well except for this and `ITEM_HAND_END + ITEM_HOLY_GLASSES;`. I'm not sure what the cleanest solution is. It seems like there's many more places where two enums work well, than places like this where it doesn't work well. So maybe that's the best choice.

Reply:
This is what @Xeeynamo suggested previously, not sure if we want to have a HAND(x) (x - ITEM_HAND_END) macro for the Holy Glasses but it seems that's the only occurrence of an item being passed as a parameter in the project for now, we can probably leave it like this until we find more evidence. For now unifying the arrays and using overlapping enums seems to make the more sense, IMO he was totally right.

Reply:
The LIB overlay might provide more hints to this in the future.